src/app/features/ledger/flows/stacks-tx-signing/ledger-sign-stacks-tx-container.tsx

```diff
@@ -129,7 +130,15 @@ function LedgerSignStacksTxContainer() {
           signedTx,
         });
       } catch (e) {
-        ledgerNavigate.toBroadcastErrorStep(isError(e) ? e.message : 'Unknown error');
+        const error = isError(e) ? e.message : 'Unknown error';
+        void analytics.track('ledger_transaction_publish_error', {
```

Comment:
I added analytics to errors that bring us to the broadcast error screen. I am unsure what we should use analytics for though.

- should only use them for events and leave errors to sentry?
- should I have access to segment?